lib/chat/key_share.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Chat.KeyShare do
+  @moduledoc "Manipulate social sharing keys"
+
+  alias Chat.Identity
+  alias Chat.{Actor, ChunkedFiles, ChunkedFilesMultisecret}
+  alias Chat.Upload.UploadKey
+  alias Chat.{Dialogs, Dialogs.Dialog, Messages}
+
+  alias ChatWeb.MainLive.Page
+
+  def generate_key_shares({me, rooms, users}) do
+    share_count = Enum.count(users)
+    base_key = Actor.new(me, rooms, %{}) |> Actor.to_encrypted_json("") |> Base.encode64()
```

Comment:
We are sharing user identity only. Rooms would be in his `My notes`

Reply:
rooms removed from key